models/tmp/stg_facebook_ads__account_history_tmp.sql

```diff
@@ -1 +1,10 @@
-select * from {{ var('account_history') }}
+{{
+    fivetran_utils.union_data(
+        table_identifier='account_history',
+        database_variable='facebook_ads_database',
+        schema_variable='facebook_ads_schema',
+        default_database=target.database,
+        default_schema='facebook_ads',
+        default_variable='account_history'
```

Comment:
When building out the social media roll up package we found a way to integrate a custom variable for the union_schema and union_databases if this feature is used across packages in the same dbt_project. Would you be able to add the `union_schema_variable` and `union_database_variable` for these `_tmp` models.

[Here](https://github.com/fivetran/dbt_facebook_pages_source/blob/master/models/tmp/stg_facebook_pages__daily_page_metrics_total_tmp.sql#L1-L12) is an example of how we do this within the dbt_facebook_pages_source package. I would envision the variable names to be `facebook_ads_union_schemas` and `facebook_ads_union_databases` respectively.

Let me know if you have any questions on this!

Reply:
Thank you! I don't have access to my dev machine this week, is it okay if I add this next Monday?

Reply:
Of course that is perfect. Thanks again!!

Reply:
Implemented, tested locally (without the social media roll-up)!